src/common/services/i18next.js

```diff
@@ -1,4 +1,4 @@
-import i18n from 'i18next';
+import i18next from 'i18next';
```

Comment:
The service itself is for internationalisation, that's why it was i18n. Why rename?
Similarly we don't have axiosClient, but apiClient. The axios as i18next here are just implementation details, no?

Reply:
Yea i was thinking about this exactly. The difference is that apiclient is providing usual methods such as get post put like any other apiclient and so the axios is just implementation detail. The same could be achieved with api-souce, get fetch or anything. However i18next is really specific, we couldnt really just swap it with react-intl. so thats why i named it. But i Am happy to revert it, i dont have a strong opinion here

Reply:
Yeah, this way is better, as it would be costly to hide it behind abstract i18n service.